content/en/getting_started/opentelemetry/_index.md

```diff
@@ -0,0 +1,277 @@
+
+1. Run the application from the `calendar/` folder:
+   ```
+   docker compose -f deploys/docker/docker-compose-otel.yml up
+   ```
+
+   This command creates a Docker container with the OpenTelemetry Collector and the Calendar service.
+
+2. Test the Calendar application by running the following from another terminal window:
+   ```
+   curl localhost:9090/calendar 
+   ```
+
+   You should get a response like:
+   ```
+   {“date":"2022-12-30"}
+   ```
+
+With each call, the Calendar application metrics, traces, and logs, which are forwarded to the OpenTelemetry Collector, the Datadog Exporter, and finally the Datadog backend. 
+
+Note: The Calendar application uses the probabilistic sampler processor, so only 30% of traces sent through the application reach the target backend. Run the curl command several times to ensure at least one trace exports to the Datadog backend. 
+
+## Exploring observability data in Datadog
+
+Now, you can use Datadog to explore the Calendar application's observability data.   
+
+*Note: It may take a few minutes for your trace data to appear in Datadog.*
+
+### Exploring Runtime and Infrastructure Metrics
+
+View runtime and infrastructure metrics to visualize, monitor, and measure the performance of your applications, hosts, containers, and processes.
+
+1. Hover over **APM** in the left-side panel and select **Service Catalog**.
+2. Hover over the `calendar-otel` service and select **Full Page**.
+3. Scroll to the bottom panel and select:
+
+   * **Infrastructure Metrics** to see your Docker container metrics, such as CPU and memory usage.
+   * **JVM Metrics** to see runtime metrics, such as heap usage and thread count. 
+
+   TODO: insert screenshot
+
+### Exploring Logs
+
+View logs to monitor and troubleshoot application and system operations.
+
+Select **Logs** in the left-side panel to explore logs generated by the Calendar application. 
+
+TODO: insert screenshot
+
+### Exploring Traces
+
+View traces and spans to observe the status and performance of requests processed by your application.
+
+1. Hover over **APM** in the left-side panel and select **Traces**.
+2. Find the **Service** section in the filter menu on the left-hand side, and select the `calendar-otel` service to list all `calendar-otel` traces:
+
+   TODO: insert screenshot
+
+3. [Explore your `calendar-otel` traces][8]. 
+
+   To start, click on a trace to open the trace side panel and find more details about the trace and its spans. For example, the [Flame Graph][9] captures how much time was spent in each component of the Calendar execution path:
+
+   TODO: insert screenshot
+
+4. Notice that you can select **Infrastructure**, **Metrics**, or **Logs** in the bottom panel to correlate your trace with other observability data. 
+
+   TODO: insert screenshot
+
+   You can also take the inverse approach: for example, in the **Logs** section of Datadog, you can click on a log and correlate it back to the respective trace.
```

Comment:
```suggestion
```
Not sure this is necessary to include. I think one way is sufficient, but let me know if you disagree.